### PR TITLE
Organized all resources into categories + added resources

### PR DESCRIPTION
**Pull request by shelleyolivia**
Hey there - I reorganized the resources into four categories to help readers better navigate all these awesome resources. I separated out general tutorials and NFT / smart contract tutorials as there were so many of the latter and I find that most developers search for the latter too when they're learning. I work at Alchemy and I also added our newly launched Alchemy University resource (completely free, with over 50k+ signups). To help round out some white space in the list, I also added in some basic Ethereum tutorials that was more tutorial and practical that gets good feedback from our developers as being a good starting part. Lmk of any questions, thanks for reviewing!

### CODE REVIEW

File: docs/develop/web3-development.md [hunk 1]
```diff
@@ -13,16 +13,37 @@ keywords:
...
+- [Hello World, smart contract tutorial](https://docs.alchemy.com/alchemy/tutorials/hello-world-smart-contract)
 - [Deploy a smart contract using REMIX IDE](https://www.quicknode.com/guides/solidity/how-to-deploy-a-smart-contract-on-matic-polygon)
 - [How to create a token (ERC20)](https://www.quicknode.com/guides/solidity/how-to-create-and-deploy-an-erc20-token)
-- [Integrating IPFS with Ethereum](https://www.quicknode.com/guides/web3-sdks/how-to-integrate-ipfs-with-ethereum)
-- [Hello World Smart Contract](https://docs.alchemy.com/alchemy/tutorials/hello-world-smart-contract)
-- [How to Create an NFT](https://docs.alchemy.com/alchemy/tutorials/how-to-create-an-nft)
+- [How to create an NFT](https://docs.alchemy.com/alchemy/tutorials/how-to-create-an-nft)
+
+**General Tutorials
+- [Full stack dapp tutorial series](https://kauri.io/#collections/Full%20Stack%20dApp%20Tutorial%20Series/full-stack-dapp-tutorial-series-intro/)
```
Comment (ashucoder9) [inline docs/develop/web3-development.md:28]:
```suggestion
- [Full stack dApp tutorial series](https://kauri.io/#collections/Full%20Stack%20dApp%20Tutorial%20Series/full-stack-dapp-tutorial-series-intro/)
```

File: docs/develop/web3-development.md [hunk 2]
```diff
@@ -13,16 +13,37 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-wiki.png
 ---
 
-- [Full stack dapp tutorial series](https://kauri.io/#collections/Full%20Stack%20dApp%20Tutorial%20Series/full-stack-dapp-tutorial-series-intro/)
-- [Web3 developer Stack](https://www.quicknode.com/guides/web3-sdks/the-web3-developer-stack)
+**Learn Web3 Programs
+- [Alchemy University (free)](https://university.alchemy.com)
```
Comment (ashucoder9) [inline docs/develop/web3-development.md:17]:
```suggestion
- [Alchemy University (free)](https://university.alchemy.com)
- [Polygon University](https://university.polygon.technology/)
```

File: docs/develop/web3-development.md [hunk 3]
```diff
@@ -13,16 +13,37 @@ keywords:
 image: https://wiki.polygon.technology/img/polygon-wiki.png
 ---
 
-- [Full stack dapp tutorial series](https://kauri.io/#collections/Full%20Stack%20dApp%20Tutorial%20Series/full-stack-dapp-tutorial-series-intro/)
-- [Web3 developer Stack](https://www.quicknode.com/guides/web3-sdks/the-web3-developer-stack)
+**Learn Web3 Programs
```
Comment (ashucoder9) [inline docs/develop/web3-development.md:16]:
```suggestion
## Learn Web3 Programs
```

File: docs/develop/web3-development.md [hunk 4]
```diff
@@ -13,16 +13,37 @@ keywords:
...
+- [Alchemy University (free)](https://university.alchemy.com)
+- [ConsenSys Academy Developer Program On-Demand course](https://consensys.net/academy/ondemand/)
+- [Dapp University](https://www.youtube.com/channel/UCY0xL8V6NzzFcwzHCgB8orQ)
+
+**Smart Contract & Token Tutorials
```
Comment (ashucoder9) [inline docs/develop/web3-development.md:21]:
```suggestion
## Smart Contract & Token Tutorials
```

File: docs/develop/web3-development.md [hunk 5]
```diff
@@ -13,16 +13,37 @@ keywords:
...
+- [Hello World, smart contract tutorial](https://docs.alchemy.com/alchemy/tutorials/hello-world-smart-contract)
 - [Deploy a smart contract using REMIX IDE](https://www.quicknode.com/guides/solidity/how-to-deploy-a-smart-contract-on-matic-polygon)
 - [How to create a token (ERC20)](https://www.quicknode.com/guides/solidity/how-to-create-and-deploy-an-erc20-token)
-- [Integrating IPFS with Ethereum](https://www.quicknode.com/guides/web3-sdks/how-to-integrate-ipfs-with-ethereum)
-- [Hello World Smart Contract](https://docs.alchemy.com/alchemy/tutorials/hello-world-smart-contract)
-- [How to Create an NFT](https://docs.alchemy.com/alchemy/tutorials/how-to-create-an-nft)
+- [How to create an NFT](https://docs.alchemy.com/alchemy/tutorials/how-to-create-an-nft)
+
+**General Tutorials
```
Comment (ashucoder9) [inline docs/develop/web3-development.md:27]:
```suggestion
## General Tutorials
```

File: docs/develop/web3-development.md [hunk 6]
```diff
@@ -13,16 +13,37 @@ keywords:
...
-- [ConsenSys Academy Developer Program On-Demand course](https://consensys.net/academy/ondemand/)
+
+**Basics of Ethereum Development
+- [Web3 developer stack](https://www.quicknode.com/guides/web3-sdks/the-web3-developer-stack)
+- [Integrating IPFS with Ethereum](https://www.quicknode.com/guides/web3-sdks/how-to-integrate-ipfs-with-ethereum)
 - [What is Ethereum?](https://blockgeeks.com/guides/ethereum/)
-- [Read Mastering Ethereum](https://github.com/ethereumbook/ethereumbook)
-- [OpenZeppelin Learn Docs](https://docs.openzeppelin.com/learn/)
+- [Mastering Ethereum](https://github.com/ethereumbook/ethereumbook)
+- [OpenZeppelin learn docs](https://docs.openzeppelin.com/learn/)
```
Comment (ashucoder9) [inline docs/develop/web3-development.md:36]:
```suggestion
- [OpenZeppelin Learn Docs](https://docs.openzeppelin.com/learn/)
```

**Comment by shelleyolivia** [comment]
Thanks for the quick reviews, you're on it! Commited the suggestions, definitely let me know if there's anything else 👍 @ashucoder9

**Review by ashucoder9 — APPROVED** [review]
Thanks for the updates @shelleyolivia 💜

**Pull request merged**